.github/workflows/template.module.yml

```diff
@@ -25,90 +25,115 @@ env:
   TOKEN_NAMEPREFIX: '${{ secrets.TOKEN_NAMEPREFIX }}'
 
 jobs:
+  # #########################
+  # #   Static validation   #
+  # #########################
+  # job_module_static_validation:
+  #   name: 'Static validation'
+  #   runs-on: ubuntu-20.04
+  #   if: (fromJson(inputs.workflowInput)).staticValidation == 'true'
+  #   steps:
+  #     - name: 'Checkout'
+  #       uses: actions/checkout@v3
+  #       with:
+  #         fetch-depth: 0
+  #     - name: Set environment variables
+  #       uses: ./.github/actions/templates/setEnvironmentVariables
+  #       with:
+  #         variablesPath: ${{ env.variablesPath }}
+  #     - name: 'Run tests'
+  #       uses: ./.github/actions/templates/validateModulePester
+  #       with:
+  #         modulePath: '${{ inputs.modulePath }}'
+  #         moduleTestFilePath: '${{ env.moduleTestFilePath }}'
+
   #########################
-  #   Static validation   #
+  #   PSRule validation   #
   #########################
-  job_module_static_validation:
-    runs-on: ubuntu-20.04
-    name: 'Static validation'
-    if: (fromJson(inputs.workflowInput)).staticValidation == 'true'
-    steps:
-      - name: 'Checkout'
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Set environment variables
-        uses: ./.github/actions/templates/setEnvironmentVariables
-        with:
-          variablesPath: ${{ env.variablesPath }}
-      - name: 'Run tests'
-        uses: ./.github/actions/templates/validateModulePester
-        with:
-          modulePath: '${{ inputs.modulePath }}'
-          moduleTestFilePath: '${{ env.moduleTestFilePath }}'
-
-  #############################
-  #   Deployment validation   #
-  #############################
-  job_module_deploy_validation:
+  job_psrule_test:
+    name: 'PSRule pre-flight validation'
```

Comment:
Other ideas could be
- `PSRule check`
- `PSRule`
- `PSRule -` (though this doesn't go well with the matrix header which would be `Matrix: Module / PSRule -`)

Reply:
Fine, but what about other validation jobs? Doesn't this apply to them as well? I'm not looking forward to changing those since we would probably need to update wiki and diagrams as well

Reply:
Yes and no. For the 'rendered' visuals on the right there isn't much we can do in any case - so as far as I'm concerned we can disregard that.
But if we can make it work on the left (the Jobs list) that would be great. There we at least have a chance.

Reply:
Would PSRule validation do the work on the left?

Reply:
I think so, yes

Reply:
PSRule validation it is